test/models/service_test.rb

```diff
@@ -19,8 +19,14 @@
 
 module Models
   class ServiceTest < Minitest::Test
-    def test_skills
+    def around
+      Models.delete_all
+      yield
+    ensure
       Models.delete_all
+    end
```

Comment:
If we need them here, don't we need this in every test?

Reply:
Most of other tests use Models::Vrp.create which clear by himself the Models store.

We usually only need this for model tests. Otherwise we need a custom class which inherits from Minitest::Test and from all our test classes needs to inherit.

We can also only inherit from this custom class for models tests. 

What to you think ?

Reply:
If the other tests are okay, we can leave it like this at the moment but I like your inheritance idea only for the group of tests who need it.